dictionaries/dict.json

```diff
@@ -101130,7 +101130,7 @@
 "STEUBG/*ER": "sticker",
 "STEUBG/-G": "sticking",
 "STEUBG/-G/RAOUPBD": "sticking around",
-"STEUBG/-S": "sticks",
```

Comment:
@JRJurman `"STEUBG/-S": "sticks",` is still valid so instead of deleting this entry, we can keep `"STEUBG/-S": "sticks",` and add `"STEUBGS": "sticks",`.

Reply:
oh shoot! I can re-add that 👍